core/storage/mpool/mpool.cpp

```diff
@@ -88,7 +90,14 @@ namespace fc::storage::mpool {
       if (apply.receipt.exit_code != vm::VMExitCode::kOk) {
         return apply.receipt.exit_code;
       }
-      // TODO: paych.collect
+      if (msg.method
+          == vm::actor::builtin::v0::payment_channel::Collect::Number) {
+        auto matcher{vm::toolchain::Toolchain::createAddressMatcher(
+            vm::version::getNetworkVersion(head->height()))};
+        if (matcher->isPaymentChannelActor(actor.code)) {
+          apply.receipt.gas_used += 76000;
```

Comment:
Please, add constant for 76000.

Reply:
It will be great if you can add a reference where the number comes from.

Reply:
https://github.com/filecoin-project/lotus/blob/191a05da4872bf9849f178e6db5c0d6e87d05baa/node/impl/full/gas.go#L281

Reply:
Fixed